integration/rails4_app/app.yaml.example

```diff
@@ -0,0 +1,5 @@
+runtime: ruby
+vm: true
```

Comment:
Since "env: flex" is currently not compatible with Google::Cloud::Core::Environment, just update "vm: true" to "env: flex" in this PR will break the tests.

I think it makes more sense to change these "vm: true" to "env: flex" in a separate PR, where I can modify Google::Cloud::Core::Environment to match the new GAE Flex environment.